[snmp] Exceptions on startup

After updating my configuration from SNMP1 to SNMP(2) binding I get the following exceptions on starting Openhab. The SNMP items seems to work correctly. I am running OpenHAB 2.5.0 M3

```
[ERROR] [ome.core.thing.link.ThingLinkManager] - Exception occurred while informing handler: null
java.lang.NullPointerException: null
	at org.snmp4j.MessageDispatcherImpl.getTransport(MessageDispatcherImpl.java:225) ~[?:?]
	at org.snmp4j.MessageDispatcherImpl.sendPdu(MessageDispatcherImpl.java:446) ~[?:?]
	at org.snmp4j.Snmp.sendMessage(Snmp.java:1087) ~[?:?]
	at org.snmp4j.Snmp.send(Snmp.java:1057) ~[?:?]
	at org.snmp4j.Snmp.send(Snmp.java:1041) ~[?:?]
	at org.openhab.binding.snmp.internal.SnmpServiceImpl.send(SnmpServiceImpl.java:133) ~[?:?]
	at org.openhab.binding.snmp.internal.SnmpTargetHandler.handleCommand(SnmpTargetHandler.java:100) ~[?:?]
	at org.eclipse.smarthome.core.thing.binding.BaseThingHandler.channelLinked(BaseThingHandler.java:229) ~[?:?]
	at org.eclipse.smarthome.core.thing.link.ThingLinkManager.lambda$0(ThingLinkManager.java:266) ~[?:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:?]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180) [?:?]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293) [?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:?]
	at java.lang.Thread.run(Thread.java:748) [?:?]
```

--- a/bundles/org.openhab.binding.snmp/src/main/java/org/openhab/binding/snmp/internal/SnmpTargetHandler.java
+++ b/bundles/org.openhab.binding.snmp/src/main/java/org/openhab/binding/snmp/internal/SnmpTargetHandler.java
@@ -96,11 +96,9 @@ public class SnmpTargetHandler extends BaseThingHandler implements ResponseListe
 
     @Override
     public void handleCommand(ChannelUID channelUID, Command command) {
-        if (target.getAddress() == null) {
-            if (!renewTargetAddress()) {
-                logger.info("failed to renew target address, can't process '{}' to '{}'.", command, channelUID);
-                return;
-            }
+        if (target.getAddress() == null &&!renewTargetAddress()) {
+            logger.info("failed to renew target address, can't process '{}' to '{}'.", command, channelUID);
+            return;
         }
 
         try {

--- a/bundles/org.openhab.binding.snmp/src/main/java/org/openhab/binding/snmp/internal/SnmpTargetHandler.java
+++ b/bundles/org.openhab.binding.snmp/src/main/java/org/openhab/binding/snmp/internal/SnmpTargetHandler.java
@@ -96,6 +96,13 @@ public class SnmpTargetHandler extends BaseThingHandler implements ResponseListe
 
     @Override
     public void handleCommand(ChannelUID channelUID, Command command) {
+        if (target.getAddress() == null) {
+            if (!renewTargetAddress()) {
+                logger.info("failed to renew target address, can't process '{}' to '{}'.", command, channelUID);
+                return;
+            }
+        }
+
         try {
             if (command instanceof RefreshType) {
                 SnmpInternalChannelConfiguration channel = readChannelSet.stream()

